src/library/dao/trackdao.cpp

```diff
@@ -967,19 +993,20 @@ bool TrackDAO::onPurgingTracks(
     }
     {
         // Remove location from track_locations table
-        FwdSqlQuery query(m_database, QString(
-                "DELETE FROM track_locations "
-                "WHERE location in (%1)").arg(
-                        SqlStringFormatter::formatList(m_database, locations)));
+        FwdSqlQuery query(m_database,
+                QString("DELETE FROM track_locations "
```

Comment:
QStringLiteral

Reply:
I will undo all unrelated changes and force push.